app/views/incomeSources/add/IncomeSourcesAccountingMethod.scala.html

```diff
@@ -51,6 +52,14 @@
     </ul>
 }
 
+@hintContent = {
+    @if(incomeSourcesType == SelfEmployment.key) {
+        <br><div>@getMessage("hint")</div></br><div>@detailsDropdown(getMessage("example"), detailsContent)</div>
```

Comment:
I would assume that this part is not complete as you calling context resource called "example"

Reply:
getMessage("example") appends to the prefix (s"$prefix.$key", args: _*) for incomeSources.add.FP.AccountingMethod.example. I did it correctly?